lib/linux_admin/service.rb

```diff
@@ -8,7 +8,6 @@ def self.service_type(reload = false)
     end
 
     class << self
-      private
```

Comment:
Not sure why this was removed?  I think this is intentional as the alias_method methods will be private, and that is no longer the case.

Reply:
rubocop reported `Lint/UselessAccessModifier: Useless private access modifier`.

This script would seem to indicate that `private` doesn't work on aliases (?):

```ruby
class Foo
  class << self
    private
    alias_method :orig_new, :new
  end
end

p Foo.orig_new # works
```

Reply:
I think I've ended up resorting to just straight up `undef`-ing the `new` method when I wanted this behavior. Or used the `singleton` library (I think).

Reply:
Interesting...I think private works with alias, but perhaps not alias_method ?

Either way, it is expected to be private so you could just do

```ruby
alias_method :orig_new, :new
private :orig_new
```

Reply:
Ok, updated.